src/v/pandaproxy/rest/proxy.h

```diff
@@ -14,6 +14,7 @@
 #include "cluster/fwd.h"
 #include "pandaproxy/rest/configuration.h"
 #include "pandaproxy/server.h"
+#include "pandaproxy/sharded_client_cache.h"
```

Comment:
`sharded_client_cache` only ever seems to be used by ref, so a forward declaration is sufficient.

Reply:
Forward declaration led to an undefined compile error. I'll keep the include in for now.